.travis.yml

```diff
@@ -24,10 +24,10 @@ before_install:
   - micromamba config set always_yes true
   - micromamba config set changeps1 false
   - micromamba config set show_channel_urls true
-  - eval "$(micromamba shell hook --shell=bash)"
 
 install:
   - micromamba env create --file ci/environment.yml
+  - eval "$(micromamba shell hook --shell=bash)"
```

Comment:
This commit only differs from the ultimate solution by sourcing ~/.bashrc in the install phase. So it appears that the explicit sourcing is necessary / not carried out by default.